### PR TITLE
add printOption to help

### PR DESCRIPTION
**Pull request by UplinkCoder**

### CODE REVIEW

File: src/dfmt/main.d [hunk 1]
```diff
@@ -157,6 +157,17 @@ else
     }
 }
 
+private string printOptions(E)() if(is(E == enum)) {
```
Comment (Hackerpilot) [inline src/dfmt/main.d]: This function doesn't do any printing.

**Comment by UplinkCoder** [comment]
I wanted to stay consistent with the naming of printHelp

**Comment by UplinkCoder** [comment]
optionToString maybe ?

**Comment by Hackerpilot** [comment]
`optionToString` works.

**Comment by UplinkCoder** [comment]
fixed

**Pull request merged**